guide/english/book-recommendations/javascript/index.md

```diff
@@ -30,6 +30,10 @@ A collection of articles covering the basics (core language and working with a b
 - [Online](https://javascript.info/)
 - [E-book](https://javascript.info/ebook)
 
+#### Secrets of the JavaScript ninja (2e)
+ Targeted to at least intermediate JS programmers, this book covers in depth features of Javascript in ES6 version (among other subjects).
+- [Amazon] (https://www.amazon.com/Secrets-JavaScript-Ninja-Second-Resig/dp/1617292850)
```

Comment:
Please can you correct the link?
- no space between the bracket and the first parentheses
- Secret Java Ninja in brackets